app/controllers/api/scores_controller.rb

```diff
@@ -15,6 +15,19 @@ def user_feed
       render json: response.to_json
     end
 
+    def golfers
+      scores = Score.all.order(played_at: :desc, id: :desc).includes(:user).filter {
```

Comment:
This can be simplified to `scores = Score.where(user_id: params[:id]).includes(:user).order(played_at: :desc, id: :desc)`